src/fragments/lib/graphqlapi/graphql-from-node.mdx

```diff
@@ -150,84 +147,60 @@ General information
+
 ```javascript
-const https = require('https');
-const AWS = require("aws-sdk");
-const urlParse = require("url").URL;
-const appsyncUrl = process.env.API_<YOUR_API_NAME>_GRAPHQLAPIENDPOINTOUTPUT;
-const region = process.env.REGION;
-const endpoint = new urlParse(appsyncUrl).hostname.toString();
-const graphqlQuery = require('./query.js').mutation;
-const apiKey = process.env.API_<YOUR_API_NAME>_GRAPHQLAPIKEYOUTPUT;
+const AppSyncClient = require('appsync-client').default;
+const gql = require('graphql-tag');
+const config = {
+  apiUrl: process.env.API_ < YOUR_API_NAME > _GRAPHQLAPIENDPOINTOUTPUT
+};
+const client = new AppSyncClient(config);
 
-exports.handler = async (event) => {
-    const req = new AWS.HttpRequest(appsyncUrl, region);
+const { createTodo } = require('./graphql.js');
 
+exports.handler = async (event) => {
+  try {
     const item = {
-        input: {
-            name: "Lambda Item",
-            description: "Item Generated from Lambda"
-        }
+      name: 'Lambda Item',
+      description: 'Item Generated from Lambda'
     };
 
-    req.method = "POST";
-    req.path = "/graphql";
-    req.headers.host = endpoint;
-    req.headers["Content-Type"] = "application/json";
-    req.body = JSON.stringify({
-        query: graphqlQuery,
-        operationName: "createTodo",
-        variables: item
-    });
-
-    if (apiKey) {
-        req.headers["x-api-key"] = apiKey;
-    } else {
-        const signer = new AWS.Signers.V4(req, "appsync", true);
-        signer.addAuthorization(AWS.config.credentials, AWS.util.date.getDate());
-    }
-
-    const data = await new Promise((resolve, reject) => {
-        const httpRequest = https.request({ ...req, host: endpoint }, (result) => {
-            let data = "";
-
-            result.on("data", (chunk) => {
-                data += chunk;
-            });
-
-            result.on("end", () => {
-                resolve(JSON.parse(data.toString()));
-            });
-        });
-
-        httpRequest.write(req.body);
-        httpRequest.end();
+    const { data } = await client.request({
+      query: gql(createTodo),
+      variables: {
+        input: item
+      }
     });
 
     return {
-        statusCode: 200,
-        body: data
+      statusCode: 200,
+      body: JSON.stringify(data.createTodo),
+      headers: {
+        'Access-Control-Allow-Origin': '*'
+      }
     };
+  } catch (error) {
+    console.log('Something went wrong in the handler:', error);
+  }
 };
 ```
 
-Finally you can define the GraphQL operation you're running, in this case the `createTodo` mutation, in a separate `query.js` file:
+Finally you can define the GraphQL operation you're running, in this case the `createTodo` mutation, in a separate `graphql.js` file:
```

Comment:
You can also define which GraphQL operation you're running. In this example, we are going to be creating a `createTodo` mutation within a separate `graphql.js` file.